unstructured/ingest/cli/cli.py

```diff
@@ -15,10 +17,16 @@ def get_cmd() -> click.Command:
     to the main command as nested subcommands.
     """
     cmd = ingest
+    src_dict = {s.name: s for s in src}
+    dest_dict = {d.name: d for d in dest}
+    for s in src_v2:
+        src_dict[s.name] = s
+    for d in dest_v2:
+        dest_dict[d.name] = d
```

Comment:
got it, so we just defer to v2 for cli if there exists a v2 command

Reply:
Yep, at least right now as soon as the new connector exists, it replaces the old one.

Reply:
Yea, this is actually pretty nice because it means that nothing breaks as far as users know. Also means we can add deprecations warnings for the Python users for whom it will break once we switch over.

Reply:
Maybe worth doing that for the ones that you're bumping here? And probably bump the docs for the respective connectors?

Reply:
Where do we manage the docs for connectors now? I thought that changed recently and not sure if that's still being managed by this repo.

Reply:
good question! so, we now host the docs themselves in this repo: https://github.com/Unstructured-IO/docs 

and it pulls source code snippets from here: https://github.com/Unstructured-IO/unstructured/tree/main/docs/source/ingest/source_connectors/code
and
https://github.com/Unstructured-IO/unstructured/tree/main/docs/source/ingest/destination_connectors/code